karina-lab10/public/scripts/articleView.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+var app = app || {module};
```

Comment:
var app = app || {}

all we want to do is set app equal to itself if it already exists. If it does not exist  we want to define it as an empty object. App will hold the contents of all of our js files(variables like Article.all and methods like articleView.populateFilters)